src/main/resources/application.properties

```diff
@@ -454,6 +454,9 @@ spring.activemq.user=
 # Login password of the broker.
 spring.activemq.password=
 
+# Cap the amount of activities sent per message to prevent memory issues
+activemq.max.message.concept-activities=200000
```

Comment:
@markrynbeek  We should add this new config in the Authoring Traceability section with matching prefix.